libs/asset-viewer/src/lib/state/map-control/map-control.actions.ts

```diff
@@ -0,0 +1,4 @@
+import { createAction } from '@ngrx/store';
+
```

Comment:
I'd prefer to just delete the [`map-control.effects.ts`]([libs/asset-viewer/src/lib/state/map-control/map-control.effects.ts](https://github.com/swisstopo/swissgeol-assets-suite/pull/274/files#diff-1045769abb100ce13eae62535b7531a8d2f26f50153c0c2c8ff83f2320417572)) file instead of having it empty.